### ISSUE TITLE
UI/UX Design for main screen

### ISSUE THREAD

**Issue by nruffilo** [nruffilo/YouTodoProj#6]
Need a main screen that will display the basic User attributes, with a "create quest" button, "create epic" and then a list of open quests / epics.  It needs to be mobile friendly as well as work on a desktop.

**Comment by mariel-delagarza** [comment]
Are we looking to build a design that's responsive/mobile-first?

**Comment by nruffilo** [comment]
Yes, everything will be designed mobile first.  My goal is to keep things "simple" so that it's easy to have things be mobile and have a clear UI/UX.  Also, my hope is to keep things feature-simple, so that everything can always be accessible.
